docs/static/images/logo.svg

```diff
@@ -0,0 +1,271 @@
+             style="fill:url(#linearGradient116);fill-opacity:1;fill-rule:nonzero;stroke:none"
+             id="path118" /></g></g><g
+         id="g120"><g
+           id="g122"
+           clip-path="url(#clipPath126)"><path
+             d="m 7038.47,6487.92 c 0,-54.7 -8.39,-107.67 -24.14,-157.2 -33.76,-106.68 -101.28,-198.58 -189.97,-262.89 -25.38,-18.23 -52.48,-34.49 -81.31,-48.29 -74.16,176.42 -174.69,339.02 -296.4,483.17 -0.25,0.48 -0.49,0.48 -0.99,0.98 31.79,77.61 82.05,145.86 145.12,199.57 87.47,73.92 199.57,119.5 322.53,122.45 78.1,-90.66 125.16,-208.68 125.16,-337.79"
+             style="fill:url(#linearGradient132);fill-opacity:1;fill-rule:nonzero;stroke:none"
+             id="path134" /></g></g><g
+         id="g136"><g
+           id="g138"
+           clip-path="url(#clipPath142)"><path
+             d="m 5413.55,4557.25 c 0,-283.09 -170.49,-526.28 -414.41,-632.96 -121.97,53.22 -225.69,140.93 -298.62,250.33 -73.18,109.39 -115.81,240.96 -115.81,382.63 0,4.93 0,9.85 0.26,14.54 123.92,64.55 264.86,101.01 414.17,101.01 149.55,0 290.49,-36.46 414.16,-101.01 0.25,-4.93 0.25,-9.61 0.25,-14.54"
+             style="fill:url(#linearGradient150);fill-opacity:1;fill-rule:nonzero;stroke:none"
+             id="path152" /></g></g><g
+         id="g154"><g
+           id="g156"
+           clip-path="url(#clipPath160)"><path
+             d="m 5965.71,4557.25 c 0,-158.68 -53.72,-305.28 -144.14,-421.82 0,-0.23 0,-0.23 0,-0.23 -126.15,-163.35 -323.99,-268.33 -546.23,-268.33 -98.32,0 -191.44,20.45 -276.2,57.42 243.92,106.68 414.41,349.87 414.41,632.96 0,4.93 0,9.61 -0.25,14.54 -5.65,276.93 -174.43,513.71 -414.16,618.42 -239.73,-104.71 -408.27,-341.49 -414.17,-618.42 -0.26,-4.69 -0.26,-9.61 -0.26,-14.54 0,-141.67 42.63,-273.24 115.81,-382.63 72.93,-109.4 176.65,-197.11 298.62,-250.33 -84.52,-36.97 -177.89,-57.42 -276.2,-57.42 -222.49,0 -420.09,104.98 -546.24,268.56 -90.17,116.54 -143.89,263.14 -143.89,421.82 0,10.59 0,20.7 0.5,31.05 1.71,21.91 3.69,43.6 6.15,65.03 69.73,649.73 435.62,1210.51 959.68,1544.84 524.3,-334.33 889.93,-895.11 959.92,-1544.84 0,0 0,-0.49 0,-0.73 2.21,-21.18 4.18,-42.62 5.9,-64.3 0.49,-10.35 0.75,-20.46 0.75,-31.05"
+             style="fill:url(#linearGradient166);fill-opacity:1;fill-rule:nonzero;stroke:none"
+             id="path168" /></g></g></g></g></svg>
```

Comment:
where did you get this logo from? I think it is great! We could also ask the design team for one.

Reply:
Free, and royalty-free stuff, just the first thing that matched my search and caught my eye.

https://www.rawpixel.com/image/524731/free-illustration-vector-bear-animal-art